### DISCUSSION TITLE
A fifth technical document: the proposal?

### DISCUSSION THREAD

**Discussion by swaits** [evildmp/diataxis-documentation-framework#149] [Ideas]
Much of what I write at work (which is a lot) is in the form of a proposal. They're supremely technical documents, but they're intended to drive a specific discussion or a decision or alignment.

My question is: Do you think this fits into one of your four doc types already? If not, would you consider adding a fifth?

Thanks!

**Comment by evildmp** [comment]
That falls outside the framework - it belongs a different kind of documentation altogether.

Diátaxis is concerned with the kind of documentation that supports someone's work. Handbook, or user manual kind of documentation. 

Proposals, design specifications, reports, compliance statements and other kinds of technical documentation are all important, but they're not what this is about, they have a different purpose.
